src/ast/rewriter/seq_rewriter.cpp

```diff
@@ -4515,6 +4624,11 @@ br_status seq_rewriter::mk_re_union(expr* a, expr* b, expr_ref& result) {
         result = mk_full();
         return BR_DONE;
     }
+
+    //just keep the union normalized
+    result = mk_regex_union_normalize(a, b);
+    return BR_DONE;
+
```

Comment:
dead code?

Reply:
Yes, will remove. The previous code was using rewrite rules, now a and b are assumed in normal form and are merged, 
the code after is dead code, will remove.